### PR TITLE
refactor: unify import/node-package plugins for better dx and portabilty

### PR DESCRIPTION
**Pull request by infiniteluke**
**What**:
- Unify `@emdaer/plugin-import` and `@emdaer/plugin-node-package` into `@emdaer/plugin-import`
- Move `@emdaer/meta` to plain markdown files. No more weird markdown in a template string!

**Why**:
These two plugins were being used for some of the same things. Lots of overlap and almost the same API. While plugin-node-package is more powerful, it was simply being used to import strings from a module export. plugin-import was being used to import markdown files. Now plugin-import can, run functions, receive strings, and resolve md files. I also added the ability to resolve functions that are async.

This enables use to use plain markdown files in `@emdaer/meta`. This is big because it also allows us to use gatsby for what it is made for on https://emdaer.me. We can simply use graphql to query the files inside `@emdaer/meta` now. Much better imo and makes that pattern (sharing md between readme and websites via emdaer) much more viable and clean!

**How**:
plugin-import now does everything node-package did including importing markdown files found from node modules. It still is able to import markdown files as it did before as well, so this is a BC change.

Content can be imported in a few ways, as long as it's a string
 - a local path to a markdown file, a partial of your readme. ex: `.emdaer/README/USAGE.md`. This just uses `readFile`.
 - ✨ **new** a path to a node module that contains markdown files in its pacakge. ex: `@emdaer/meta/lib/what-is-emdaer.md`. This means you can expose READMEs in a node package and import them from emdaer.
 - a path to a node module exporting a string. ex: `moduleName` in this case `moduleName` exports a string.
 - a path to a node module exporting a function that takes arguments and returns/resolves to a string ex: `moduleName`. This one is super powerful. ✨ **new** Can handle promises as well.

**Checklist**:

- [x] Documentation
- [x] Tests
- [x] Ready to be merged 
- [x] Added myself to AUTHORS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +35,7 @@
 </ul>
 <h2 id="how-emdaer-works">How emdaer works</h2>
 <p>emdaer processes template files and writes the resulting files to your project.</p>
-<p>We match <code>.emdaer/(**/*).emdaer(.md)</code> and use the captured part of each matched file to determine the path for the output.</p>
+<p>We match `.emdaer/(*_/_).emdaer(.md)` and use the captured part of each matched file to determine the path for the output.</p>
```
Comment (flipactual) [inline README.md]: Is this a prettier bug?

File: README.md [hunk 2]
```diff
@@ -24,6 +24,7 @@
 <li><a href="#license">License</a></li>
 </ul>
 <!-- tocstop -->
+<p>Hello world x3<br>Hello world x3<br>Hello world x3</p>
```
Comment (infiniteluke) [inline README.md]: Oops. This should be removed

File: .emdaer/README.emdaer.md [hunk 3]
```diff
@@ -35,46 +35,53 @@
 -->
 
 <!--emdaer-p
-  - '@emdaer/plugin-node-package'
-  - path: '@emdaer/meta/lib/README/what-is-emdaer.js'
+  - '@emdaer/plugin-import'
```
Comment (infiniteluke) [inline .emdaer/README.emdaer.md:38]: Remove

File: .emdaer/printThrice.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+module.exports = function printThrice(item) {
```
Comment (infiniteluke) [inline .emdaer/printThrice.js]: Remove file

File: README.md (hunk 1 above)
Comment (infiniteluke) [reply README.md]: kind of seems like a bug ya, though. i'm also interested in why those code tags went away

**Comment by infiniteluke** [comment]
I think this closes #93 as well.

Templates are just readmes you import via `@emdaer/plugin-import` from a node module you publish. So, you just publish a markdown file with emdaer calls in it and then use plugin import to import it in your project.

```md
<!--emdaer-p
  - '@emdaer/plugin-import'
  - path: '@emdaer/meta/src/template.md'
    runEmdaer: true
-->
```

**Comment by infiniteluke** [comment]
This needs to be either rebased as a `feat` or squashes as such. this is not a `refactor` as my commit message implies

File: .emdaer/README.emdaer.md (hunk 3 above)
Comment (flipactual) [reply .emdaer/README.emdaer.md:38]: ?

Comment (infiniteluke) [reply .emdaer/README.emdaer.md:38]: @flipactual github left the comment in the same spot, even though my commit resolved the issue. should be ok here.

File: .emdaer/README.emdaer.md [hunk 5]
```diff
@@ -35,46 +35,46 @@
...
+  - path: '@emdaer/meta/src/README/core-transforms.md'
     runEmdaer: true
 -->
 
 <!--emdaer-p
-  - '@emdaer/plugin-node-package'
-  - path: '@emdaer/meta/lib/README/core-transforms.js'
-    runEmdaer: true
+  - '@emdaer/plugin-import'
+  - path: '@emdaer/meta/src/README/adding-emdaer-to-your-project.md'
```
Comment (flipactual) [inline .emdaer/README.emdaer.md:61]: Intentional reordering?

File: flow-typed/npm/lodash.get_vx.x.x.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+ * needed.
+ */
+
+// Filename aliases
+declare module 'lodash.get/index' {
+  declare module.exports: $Exports<'lodash.get'>;
+}
+declare module 'lodash.get/index.js' {
+  declare module.exports: $Exports<'lodash.get'>;
+}
```
Comment (flipactual) [inline flow-typed/npm/lodash.get_vx.x.x.js:32]: Let's switch to Ramda

**Review by flipactual — COMMENTED** [review]
Glanced over, everything looks good, going to functionally test

File: .emdaer/README.emdaer.md (hunk 5 above)
Comment (infiniteluke) [reply .emdaer/README.emdaer.md:61]: Yes. Open to going back to what we had, but it felt natural to list the plugins/transforms after explaining what plugins/transforms do.

**Review by flipactual — APPROVED** [review]
lgtm!

File: flow-typed/npm/lodash.get_vx.x.x.js (hunk 6 above)
Comment (infiniteluke) [reply flow-typed/npm/lodash.get_vx.x.x.js:32]: I tried, but honestly gave up when I got this using `pathOr`..
screen shot 2018-04-28 at 4 51 37 pm

**Pull request merged**